go.mod

```diff
@@ -19,10 +19,12 @@ require (
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/pires/go-proxyproto v0.4.2
+	github.com/prometheus/client_golang v1.10.0 // indirect
 	github.com/sirupsen/logrus v1.7.0 // indirect
 	golang.org/x/net v0.0.0-20210119194325-5f4716e94777
-	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
 	golang.org/x/time v0.0.0-20201208040808-7e3f01d25324 // indirect
 	google.golang.org/grpc v1.35.0 // indirect
 	gotest.tools/v3 v3.0.3 // indirect
 )
+
+replace github.com/haveachin/infrared => ./
```

Comment:
What does this do?

Reply:
Oh oops, that's the replace I used so go would use my fork while testing instead of upstream, and apparently I accidentally checked it in, I'll get rid of it.

Reply:
Ah, I see